std_msgs/Header header
std_msgs/String id
geometry_msgs/Pose position
geometry_msgs/Pose velocity
geometry_msgs/Pose goal
geometry_msgs/Pose cov_pose
geometry_msgs/Pose cov_vel
geometry_msgs/Pose cov_goal
geometry_msgs/Pose force_repulsive
geometry_msgs/Pose force_goal
geometry_msgs/Pose force_wall